jina/peapods/pea.py

```diff
@@ -319,24 +320,39 @@ def run(self):
             self.post_init()
             self.loop_body()
         except RequestLoopEnd:
-            self.logger.info('break from the event loop')
+            self._message.envelope.status = jina_pb2.Envelope.Status.ERROR
```

Comment:
Changing `message` does not have any effect. The message won't propagate as the `send` is in the `loop_body`

Reply:
incorporated the change, raising pod run time exception in the last except Exception as ex clause below